src/components/project/ProjectBasicsPane.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import FormControl from "@material-ui/core/FormControl";
+import TextField from "@material-ui/core/TextField";
+import OrganizationSelect from "../../components/general/OrganizationSelect.js";
```

Comment:
Nitpick: we shouldn't add the file extension for JS imports—in other words, we can remove the `.js` suffix. This isn't a huge deal, but lets us more easily change the file extension in the future. We might do that if we switch to TypeScript as one example.

This nitpick applies to a few other places in this PR, too.